lib/lita/handlers/reviewme.rb

```diff
@@ -118,22 +111,16 @@ def mention_reviewer(response, room: get_room(response))
 
       private
 
-      def next_reviewer(room)
-        ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST)
-      end
+      def next_reviewer(room, owner = nil)
+        raise Lita::Reviewme::NoReviewer unless reviewer = ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST)
```

Comment:
This feels a bit like using an exception as control flow. The code might be clearer if you flow the code with conditions.